src/fiber/network.rs

```diff
@@ -2439,7 +2445,11 @@ impl NetworkSyncState {
 
         if should_create {
             let graph_syncer = Actor::spawn_linked(
-                Some(format!("Graph syncer to {}", peer_id)),
+                Some(format!(
+                    "Graph syncer to {} started at {:?}",
+                    peer_id,
```

Comment:
do we need to specify a name for all `spawn_linked` invocations?
I found there are some reference of invocations with name of `None`.
maybe it's helpful for debugging when panic happened.

Reply:
I think setting names for all actors is helpful. One small problem is that we have set the name before spawning any actors. For a outbound channel whose channel ID is still undetermined, it is hard for us to set a meaningful name. If adding name for all actors is desirable in this PR, I can do that.